nomad/scripts/ci_allocs.sh

```diff
@@ -77,6 +80,7 @@ function generate_run_summary() {
           --argjson started_at "$started_at" \
           --arg behaviours "${behaviours:-}" \
           --arg wind_tunnel_version "$wind_tunnel_version" \
+          --argjson holochain_build_info "$holochain_build_info" \
```

Comment:
The documentation of the `generate_run_summary()` function needs to get updated accordingly on line 47.